account_invoice_production_lot/models/account_invoice.py

```diff
@@ -31,10 +31,18 @@ class AccountInvoiceLine(models.Model):
         compute='_compute_line_lots',
     )
 
+    @api.multi
+    def _get_location_id(self):
+        self.ensure_one()
+        order = self.sale_line_ids[0].order_id
+        return order.partner_shipping_id.property_stock_customer
```

Comment:
I don't think this is the proper location to use for all cases. Why not using the location type = 'internal'?

Reply:
I had a try locally with:

```
    @api.multi
    def _compute_prod_lots(self):
        for line in self:
            quant_ids = line.move_line_ids.mapped('quant_ids').filtered(
                lambda x: x.location_id.usage == 'internal')
            line.prod_lot_ids = quant_ids.mapped('lot_id')
```
but tests are failing:
`2019-01-22 15:07:30,175 20785 ERROR test-account-invoice-reporting-93 odoo.modules.module: Module account_invoice_production_lot: 3 failures, 0 errors
`

I used that location by the same way as https://github.com/odoo/odoo/blob/10.0/addons/sale_stock/models/sale_order.py#L167